angular-viewer/adam-angular-demo/src/app/components/image-output/image-output.component.ts

```diff
@@ -38,36 +38,33 @@ export class ImageOutputComponent implements OnChanges {
 
   ngOnChanges(changes: SimpleChanges): void {
     this.reset();
-
-    for (const path_id in changes.scene_images.currentValue) {
-      this.getImage(
-        changes.scene_images.currentValue[path_id],
-        this.sceneImageArray
-      );
-    }
-    for (const path_id in changes.stroke_images.currentValue) {
-      this.getImage(
-        changes.stroke_images.currentValue[path_id],
-        this.sceneStrokeArray
-      );
-    }
-    for (const path_id in changes.stroke_graph_images.currentValue) {
-      this.getImage(
-        changes.stroke_graph_images.currentValue[path_id],
-        this.sceneStrokeGraphArray
-      );
-    }
-
+    this.sceneImageArray = changes.scene_images.currentValue;
+    this.sceneStrokeArray = changes.stroke_images.currentValue;
+    this.sceneStrokeGraphArray = changes.stroke_graph_images.currentValue;
     this.isImg = true;
   }
 
   onClickCarousel(selection: string): void {
     this.carouselSelection = selection;
   }
 
-  getImage(image_path: string, arr: Array<string>): void {
+  getImage(
```

Comment:
Did we have a reason to keep this function? I forget. It doesn't seem to be in use.

Reply:
Currently it's not in use, so I can remove it

Reply:
I think that makes sense.